setup.py

```diff
@@ -35,5 +35,5 @@
     license_files=("LICENSE",),
     packages=find_packages(include=["tico*"]),
     entry_points={"console_scripts": ["pt2-to-circle = tico.pt2_to_circle:main"]},
-    install_requires=["circle-schema", "packaging", "cffi", "torch", "pyyaml"],
+    install_requires=["circle-schema", "packaging", "cffi", "torch", "pyyaml", "tqdm"],
```

Comment:
Well it's in `experimental` directory. Do you expect it to be in strong requirement of setup file?

Reply:
```py
setup(
    ...
    extras_require={
        "experimental": ["tqdm",]
    }
)
```

Maybe..?